.circleci/config.yml

```diff
@@ -159,6 +159,7 @@ jobs:
           name: Run integration container tests
           command: |
             export PATH=$GOROOT/bin:$PATH
+            export NODE_OS=both
```

Comment:
I don't understand what this is doing. It doesn't look like we're prepared to handle `both` anywhere else in the code.

Reply:
We're no longer using it, and besides, it wouldn't allow us to add a 3rd node OS if we needed it...

Reply:
I was thinking about adding a list in the config like:
```
NODE_OSES:
  - "centos"
  - "ubuntu"
  - "centos ubuntu"
  - "ubuntu centos" (to make sure order doesn't matter)

And then have a function in the test which randomly selects one of these choices.  Thoughts?

Reply:
Just supporting a single OS as a value to the NODE_OS env var.  `export NODE_OS=both` has been removed.